manager/controlapi/service.go

```diff
@@ -257,6 +257,10 @@ func (s *Server) UpdateService(ctx context.Context, request *api.UpdateServiceRe
 		}
 		service.Meta.Version = *request.ServiceVersion
 		service.Spec = *request.Spec.Copy()
+
+		// Reset update status
+		service.UpdateStatus = nil
```

Comment:
I've suggested this, but now that I'm seeing it I'm wondering if this should be done in the orchestrator